### PR TITLE
Create token

### PR DESCRIPTION
**Pull request by alistair-singh**
Snowbridge Companion: https://github.com/Snowfork/snowbridge/pull/846
Resolves: SNO-490

### CODE REVIEW

File: polkadot-parachain/src/chain_spec/bridge_hubs.rs [hunk 1]
```diff
@@ -344,7 +346,7 @@ pub mod rococo {
 			},
 			// Ethereum Bridge
 			ethereum_inbound_queue: bridge_hub_rococo_runtime::EthereumInboundQueueConfig {
-				allowlist: Default::default(),
+				allowlist: vec![sp_core::H160([0x87, 0xd1, 0xf7, 0xfd, 0xfE, 0xe7, 0xf6, 0x51, 0xFa, 0xBc, 0x8b, 0xFC, 0xB6, 0xE0, 0x86, 0xC2, 0x78, 0xb7, 0x7A, 0x7d])],
```
Comment (vgeddes) [inline polkadot-parachain/src/chain_spec/bridge_hubs.rs]: We should be setting the allowlist via a governance call. Can you add a ticket to track this?

File: polkadot-parachain/src/chain_spec/bridge_hubs.rs [hunk 2]
```diff
@@ -344,7 +346,9 @@ pub mod rococo {
 			},
 			// Ethereum Bridge
 			ethereum_inbound_queue: bridge_hub_rococo_runtime::EthereumInboundQueueConfig {
-				allowlist: Default::default(),
+				allowlist: vec![sp_core::H160([
+					0xEE, 0x91, 0x70, 0xAB, 0xFb, 0xf9, 0x42, 0x1A, 0xd6, 0xDD, 0x07, 0xF6, 0xBD, 0xec, 0x9D, 0x89, 0xF2, 0xB5, 0x81, 0xE0,
+				])],
```
Comment (vgeddes) [inline polkadot-parachain/src/chain_spec/bridge_hubs.rs]: Rather use `hex_literal::hex!("0xabcde...").into()`

File: polkadot-parachain/src/chain_spec/bridge_hubs.rs (hunk 1 above)
Comment (alistair-singh) [reply polkadot-parachain/src/chain_spec/bridge_hubs.rs]: Resolved by https://linear.app/snowfork/issue/SNO-506.

File: parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs [hunk 3]
```diff
@@ -531,7 +536,8 @@ pub mod bridging {
 		pub DotLocation: MultiLocation =  MultiLocation::new(2, X1(GlobalConsensus(PolkadotNetwork::get())));
 
 		pub EthereumNetwork: NetworkId = NetworkId::Ethereum { chain_id: 15 };
-		pub EthereumLocation: MultiLocation = MultiLocation::new(2, X1(GlobalConsensus(EthereumNetwork::get()))); // TODO: Maybe registry address belongs here
+		pub EthereumLocation: MultiLocation = MultiLocation::new(2, X1(GlobalConsensus(EthereumNetwork::get())));
+		pub EthereumNativeTokensLocation: MultiLocation = EthereumLocation::get().pushed_with_interior(AccountKey20 { network: None, key: [0xD1, 0x84, 0xC1, 0x03, 0xF7, 0xAC, 0xC3, 0x40, 0x84, 0x7E, 0xEE, 0x82, 0xA0, 0xB9, 0x09, 0xE3, 0x35, 0x8B, 0xC2, 0x8D] }).unwrap();
```
Comment (vgeddes) [inline parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs]: Rather use the `hex_literal::hex!` macro for hex strings.

Comment (vgeddes) [inline parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs]: Also needs a comment explaining what this contract address refers too.

File: polkadot-parachain/src/chain_spec/bridge_hubs.rs [hunk 4]
```diff
@@ -344,7 +346,10 @@ pub mod rococo {
 			},
 			// Ethereum Bridge
 			ethereum_inbound_queue: bridge_hub_rococo_runtime::EthereumInboundQueueConfig {
-				allowlist: Default::default(),
+				allowlist: vec![sp_core::H160([
```
Comment (vgeddes) [inline polkadot-parachain/src/chain_spec/bridge_hubs.rs]: rather use `hex_literal::hex!` and then `into()` here.

File: polkadot-parachain/src/chain_spec/asset_hubs.rs [hunk 5]
```diff
@@ -331,6 +331,7 @@ pub fn asset_hub_kusama_local_config() -> AssetHubKusamaChainSpec {
 					get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
 					get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
 					get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+					hex!("5d6987649e0dac78ddf852eb0f1b1d1bf2be9623d81cb16c17cfa145948bb6dc").into(),
```
Comment (vgeddes) [inline polkadot-parachain/src/chain_spec/asset_hubs.rs:336]: Needs a comment explaining what this account is.

File: parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs [hunk 6]
```diff
@@ -568,7 +568,7 @@ impl pallet_bridge_relayers::Config for Runtime {
 // Ethereum Bridge
 
 parameter_types! {
-	pub const EthereumNetwork: xcm::v3::NetworkId = xcm::v3::NetworkId::Ethereum { chain_id: 1 };
+	pub const EthereumNetwork: xcm::v3::NetworkId = xcm::v3::NetworkId::Ethereum { chain_id: 15 };
```
Comment (vgeddes) [inline parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs:572]: Needs a comment explaining that this is a local test network of Ethereum.

File: polkadot-parachain/src/chain_spec/bridge_hubs.rs [hunk 7]
```diff
@@ -271,6 +272,7 @@ pub mod rococo {
 						// Ethereum Bridge Relayer Accounts
 						get_account_id_from_seed::<sr25519::Public>("BeaconRelay"),
 						get_account_id_from_seed::<sr25519::Public>("ExecutionRelay"),
+						ParaId::from(1000).into_account_truncating(),
```
Comment (vgeddes) [inline polkadot-parachain/src/chain_spec/bridge_hubs.rs:277]: Needs a comment explaining what parachain this is.

File: parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs [hunk 8]
```diff
@@ -522,16 +527,26 @@ pub mod bridging {
 	use xcm_builder::UnpaidRemoteExporter;
 
 	parameter_types! {
-		pub BridgeHubKusamaParaId: u32 = 1002;
+		pub BridgeHubKusamaParaId: u32 = 1013;
```
Comment (vgeddes) [inline parachains/runtimes/assets/asset-hub-kusama/src/xcm_config.rs:530]: Fine for now - but obviously we can't merge this into upstream cumulus. So we'll need to either introduce a new asset-hub runtime for rococo or change our bridge hub ParaId

**Review by vgeddes — APPROVED** [review]

**Pull request merged**